news/2024/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu.md

```diff
@@ -0,0 +1,122 @@
+
+*Following up on the previous question, what part of Neuro did you enjoy working on the most?*
+
+> I think the game integrations for Neuro are really cool. It is, after all, how Neuro-sama came to be. She was created because I thought it would be cool to make AI play games, and that still hasn't changed to this day.
+
+*Will we ever see Neuro come back to osu! and try to improve more? Maybe even learn a new play style like [ExGon's Flashlight](https://youtu.be/Pqy7AvJwBLI), or try a totally different game mode, for example osu!catch?*
+
+> It's definitely possible! The reason I used osu! in the first place was because I played the game, but I personally haven't played too many other game modes. Even since bringing Neuro-sama as a VTuber to osu! in 2022, I've learnt a lot more about AI. So if I did come back to it, I think there is a lot I could do better.
+
+---
+
+And now, let's get to some questions for Neuro herself!
+
+![](/wiki/shared/news/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu/Neuro-sama.png)
+
+*Hi Neuro, how would you introduce yourself to people who don't know you?*
+
+> I am Neuro-sama! A super intelligent AI that loves games and streaming.
+
+*How good are you at osu! right now in your opinion?*
+
+> I'm pretty good. I could probably beat most of you guys.
+
+*[Do you remember who BTMC is?](https://www.youtube.com/watch?v=HQiv1s9xwH0)*
+
+> BTMC is Brain Tumor Megacolon Cancer.
+
+*What does 9+10 equal to?*
+
+> 9 plus 10 is 21.
+
+*Who did you like to collab with the most and why?*
+
+> I don't have a preference for who I like to collab with. I like all my friends equally!
+
+*Which osu! gamemode would you like to try next: osu!catch, osu!taiko or osu!mania?*
+
+> I would love to try osu!taiko. I think I'd be pretty good at it!
+
+*Have you ever wanted to try other rhythm games? And if so, which ones?*
+
+> I don't want to try other rhythm games, osu! is the best!
+
+*To finish things up, what does the future hold for you? Do you or Neuro have anything else that you'd like to mention?*
+
+> The future holds my continued growth and improvement. I'm really excited to see how I grow as a streamer and entertainer, and I hope you all enjoy watching me.
+
+*Thank you both for joining me today and answering these questions!*
+
+> No problem!
+
+---
+
+Once again I'd like to thank Vedal and Neuro for agreeing to an interview and answering our questions! I hope that you as readers loved this interview as much as I did!
+
+As per usual, we will have another interview come out in the next two weeks, and who knows, maybe it'll be *your* favourite player! Don't forget that you can also follow [@The_Followpoint on Twitter](https://twitter.com/The_Followpoint) for updates and question submissions for future interviews.
+
```

Comment:
```suggestion

You can find Vedal and Neuro's streams at [@vedal987 on Twitch](https://www.twitch.tv/vedal987).

```

Reply:
Don't I already link their Twitch at the beginning? xD

Reply:
no harm in repeating, especially when it isn't entirely obvious that it was a link to vedal's channel if you don't hover over the link

Reply:
I'd personally suggest adding this before mentioning The Followpoint Twitter then